style.css

```diff
@@ -0,0 +1,226 @@
+  justify-content: space-between;
+  box-shadow: 0 20px 20px rgba(0, 0, 0, 0.2);
+}
+
+section .container .contactinfo h2{
+  font-size: 24px;
+  font-weight: 500;
+}
+
+section .container .contactinfo .info{
+  position: relative;
+  margin: 20px 0;
+}
+
+section .container .contactinfo .info li{
+  position: relative;
+  list-style: none;
+  display: flex;
+  margin: 20px 0;
+  cursor: pointer;
+  align-items: flex-start;
+}
+
+section .container .contactinfo .info li span:nth-child(1)
+{
+ width: 30px;
+ min-width: 30px;
+}
+
+section .container .contactinfo .info li span:nth-child(1) img{
+  max-width: 100%;
+  filter: invert(1);
+  opacity: 0.5;
+}
+
+section .container .contactinfo .info li span:nth-child(2)
+{
+color: #fff;
+margin-left: 10px;
+font-weight: 300;
+opacity: 0.5;
+}
+
+section .container .contactinfo .info li:hover span:nth-child(1) img,
+section .container .contactinfo .info li:hover span:nth-child(2)
+{
+  opacity: 1;
+}
+
+
+section .container .contactinfo .soc{
+  position: relative;
+  display: flex;
+}
+
+section .container .contactinfo .soc li{
+  list-style: none;
+  margin-right: 15px;
+}
+
+section .container .contactinfo .soc li a{
+text-decoration: none;
+}
+
+section .container .contactinfo .soc li a img{
+  filter: invert(1);
+  opacity: 0.5;
+}
+
+section .container .contactinfo .soc li:hover a img{
+  opacity: 1;
+}
+
+section .container .contactForm{
+  color: black;
+  position: absolute;
+  padding: 70px 50px;
+  background: #fff;
+  margin-left: 350px;
+  padding-left: 250px;
+  width: calc(100%) - 150px;
+  height: 90%;
+  box-shadow: 0 50px 50px rgba(0, 0, 0, 0.5);
+}
+
+section .container .contactForm h2{
+  color: #ee122a;
+  font-size: 24px;
+  font-weight: 500;
+}
+
+section .container .contactForm .formbox{
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  padding-top: 30px;
+}
+
+section .container .contactForm .formbox .inputbox{
+  position: relative;
+  margin: 0 0 35px 0;
+
+}
+
+section .container .contactForm .formbox .inputbox w50{
+  width: 47%;
+}
+
+section .container .contactForm .formbox .inputbox w100{
+  width: 100%
+}
+
+section .container .contactForm .formbox .inputbox input{
+  width: 100%; 
+  padding: 5px 0;
+  font-size: 18px;
+  color: #333;
+  border: none;
+  border-bottom: 1px solid #ee122a;
+  outline: none;
+}
+
+section .container .contactForm .formbox .inputbox textarea{
+  resize: none;
+  min-height: 120px;
+}
+
+section .container .contactForm .formbox .inputbox span{
+  position: absolute;
+  left: 0;
+  padding: 5px 0;
+  font-size: 18px;
+  font-weight: 300;
+  color: #333;
+  transition: 0.5s;
+  pointer-events: none;
+
+}
+
+section .container .contactForm .formbox .inputbox input:focus ~ span,
+section .container .contactForm .formbox .inputbox textarea:focus ~ span,
+section .container .contactForm .formbox .inputbox input:valid ~ span,
+section .container .contactForm .formbox .inputbox textarea:valid ~ span
```

Comment:
No need to select too much selector 2 3 are fine
ex: .inputbox input:focus ~ span,
 .inputbox textarea:focus ~ span,
 .inputbox input:valid ~ span,
.inputbox textarea:valid ~ span

this will also work fine,
same is for whole stylesheet selector's

Reply:
selectors reduced to some limits.